src/pages/Blocks/BlockDetailsPage.vue

```diff
@@ -0,0 +1,260 @@
+<script setup lang="ts">
+import { useRouter } from 'vue-router';
+import { computed, onMounted, ref } from 'vue';
+import { http } from '@/shared/api';
+import BaseContentBlock from '@/shared/ui/components/BaseContentBlock.vue';
+import { useTable } from '@/shared/lib/table';
+import BaseTable from '@/shared/ui/components/BaseTable.vue';
+import { type filterTransactionsModel as ftm, TransactionStatusFilter } from '@/features/filter-transactions';
+import BaseHash from '@/shared/ui/components/BaseHash.vue';
+import { transactionModel } from '@/entities/transaction';
+import { useWindowSize } from '@vueuse/core';
+import { useErrorHandlers } from '@/shared/ui/composables/useErrorHandlers';
+import BaseLoading from '@/shared/ui/components/BaseLoading.vue';
+import DataField from '@/shared/ui/components/DataField.vue';
+import { format } from '@/shared/lib/time';
+import TransactionStatus from '@/entities/transaction/TransactionStatus.vue';
+
+const router = useRouter();
+
+const { handleUnknownError } = useErrorHandlers();
+
+const TRANSACTION_HASH_BREAKPOINT = 1200;
+const { width } = useWindowSize();
+
+const transactionHashType = computed(() => (width.value < TRANSACTION_HASH_BREAKPOINT ? 'short' : 'full'));
+
+const blockHeight = computed(() => {
+  const height = router.currentRoute.value.params['height'];
+
+  if (typeof height === 'string') return height;
+
+  return height[0];
```

Comment:
It should never be an array. Use `invariant` to assert that the parameter should be a single string, the only acceptable value.